### PR TITLE
Sync Patch Code

### PR DESCRIPTION
**Pull request by richrace**
### What

- Sync Patch Code from UH 

### Why

- To allow us to show which case is in which patch

### CODE REVIEW

File: spec/lib/hackney/income/tenancy_prioritiser/universal_housing_criteria_spec.rb [hunk 1]
```diff
@@ -429,4 +429,38 @@
...
+      end
+    end
+
+    context 'when there is no property' do
+      it 'is nil' do
+        expect(criteria.patch_code).to be_nil
+      end
+    end
+
+    context 'when there is a property but no patch code' do
```
Comment (kjdchapman) [inline spec/lib/hackney/income/tenancy_prioritiser/universal_housing_criteria_spec.rb]: The before/context blocks are a little confusing for me. I'd expect this to be structured something like:

```ruby
context 'with an existing property reference' do
  context 'with a patch code' do; end
  context 'without a patch code' do; end
end

context "with a property reference that doesn't resolve"
end

Comment (richrace) [reply spec/lib/hackney/income/tenancy_prioritiser/universal_housing_criteria_spec.rb]: Agreed.

Comment (richrace) [reply spec/lib/hackney/income/tenancy_prioritiser/universal_housing_criteria_spec.rb]: Agreed.

Comment (kjdchapman) [reply spec/lib/hackney/income/tenancy_prioritiser/universal_housing_criteria_spec.rb]: Nice one, 1eedce7529 addresses this well :tada:

**Pull request merged**